internal/storage/v1/elasticsearch/spanstore/reader_test.go

```diff
@@ -311,36 +313,38 @@ func TestSpanReader_GetTrace(t *testing.T) {
 					{Hits: searchHits},
 				},
 			}, nil)
-		query := spanstore.GetTraceParameters{TraceID: model.NewTraceID(0, 1)}
+		query := []dbmodel.TraceID{dbmodel.TraceID(testingTraceId)}
 		trace, err := r.reader.GetTrace(context.Background(), query)
 		require.NotEmpty(t, r.traceBuffer.GetSpans(), "Spans recorded")
 		require.NoError(t, err)
 		require.NotNil(t, trace)
-
+		assert.Len(t, trace, 1)
 		expectedSpans, err := r.reader.collectSpans(hits)
 		require.NoError(t, err)
 
-		require.Len(t, trace.Spans, 1)
-		assert.EqualValues(t, trace.Spans[0], expectedSpans[0])
+		require.Len(t, trace[0].Spans, 1)
+		assert.EqualValues(t, trace[0].Spans[0], expectedSpans[0])
 	})
 }
 
 func TestSpanReader_multiRead_followUp_query(t *testing.T) {
 	withSpanReader(t, func(r *spanReaderTest) {
+		traceID1 := testingTraceId + "1"
+		traceID2 := testingTraceId + "2"
```

Comment:
you keep converting them later, do it once, here
```suggestion
		traceID1 := dbmodel.TraceID(testingTraceId + "1")
		traceID2 := dbmodel.TraceID(testingTraceId + "2")
```